examples/benchmark/README.md

```diff
@@ -0,0 +1,39 @@
+# Heartbeat Detection Benchmarks
+This example allows to reproduce the benchmarks shown in the repo's main [`README.md`](https://github.com/cbrnr/sleepecg#readme).
+
+## Usage
+To run the benchmark, (create a virtual enviroment), install the requirements using
+```
+pip install -r requirements-benchmark.txt
+```
+
+and execute
+```
+python benchmark_detectors.py <benchmark>
+```
+where `<benchmark>` is a toplevel-key in `config.yml` (see section [Configuration](#configuration)). Possible options: `runtime`, `metrics`, `rri_similarity`.
+
+Plots can be created by executing
+```
+python plot_benchmark_results.py <results.csv>
```

Comment:
```suggestion
python plot_benchmark_results.py results.csv
```

Reply:
I'd keep the `<>` since it's a placeholder for e.g. `rri_similarity__2021_08_18__12_23_07.csv`.

Reply:
Ah, got it. I thought it was the literal name.